src/checks.js

```diff
@@ -22,7 +24,17 @@ class Result {
    * See: https://api.github.com/emojis
    */
   get emoji() {
-    return `:${this.indication}_circle:`;
+    const { indication } = this;
```

Comment:
The switch statement here is a little bit idiosyncratic IMO. I'd have a data object like this:
```
const indicationEmojis = {
  [PASS]: 'green_circle',
  [WARN]: 'yellow_circle',
  [FAIL]: 'red_circle',
}
```
And then this logic would be more like:
```
const emoji = indicationEmojis[indication];
if (!emoji) throw new Error(/* msg */);
return emoji;
```
Which to me feels more idiomatic and readable. You could still use a template literal for the '_circle' part but I guess that's just a matter of preference.

Reply:
It feels less repetitive if I continue to use the template literal, but otherwise I've adopted your suggested approach: https://github.com/ably/repository-audit/pull/47/commits/d99ef10d3c44e7c93492a17e703f7012e7432425

Thanks. 🤓